demisto_sdk/commands/common/content/tests/objects/pack_objects/job/job_test/Jobs/sample-job.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
no, there it's `job-<name>.json`, but having a job with a non-standard name is useful for testing. I understand this can be confusing for anyone who looks at this code. Should I change it, and have two tests instead? (legit/non-legit name?)